frappe_social/public/js/frappe/social/pages/UserList.vue

```diff
@@ -184,8 +184,9 @@ export default {
 };
 </script>
 
-<style lang="less" scoped>
-@import "../../../../../../../frappe/frappe/public/less/variables.less";
+<style lang="less">
+@import "../../../../less/variables.less";
```

Comment:
Do we still need this if it's included in the app_css?

Reply:
border-color is used in styling, it fails as border-color not found and it also fails while building

Reply:
I see, can we add that styling into the `social.less` file to avoid the import?

Reply:
It works after adding styling to social.less

Reply:
Perfect